.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,11 +34,15 @@ merge of your pull request!
 
 <!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->
 
-- [ ] Documentation added to the
-      [docs site](https://github.com/alexkrolick/testing-library-docs)
-- [ ] I've prepared a PR for types targetting https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/testing-library__dom
+- [ ] Documentation added to the [docs site][docs]
+- [ ] I've prepared a PR for types targeting [DefinitelyTyped][types]
 - [ ] Tests
 - [ ] Ready to be merged
-      <!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->
+
+<!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->
```

Comment:
This needs to be on the line it was before so it's clear to what it's referring.